README.md

```diff
@@ -457,6 +457,34 @@ export GOOGLE_APPLICATION_CREDENTIALS="KEY_PATH"
 
 Replace `KEY_PATH` with path of the JSON file that contains your service account key.
 
+### Cloudflare R2
```

Comment:
Should mention that it should also use `s3a://`

Reply:
I have the example in `delta-sharing-server.yaml.template`, should I also include it here? Even if the section seems to be mainly focused on credentials

Reply:
Yes. A brief note should be enough. It was also mentioned above in the `s3` section.

Reply:
Added 👍